webtranslate/parameter_info_table.py

```diff
@@ -20,25 +20,26 @@
     ParameterInfo("TINYFONT",       [], None, False, True ),
     ParameterInfo("BIGFONT",        [], None, False, True ),
 
-    ParameterInfo("BLUE",           [], None, False, False ),
-    ParameterInfo("SILVER",         [], None, False, False ),
-    ParameterInfo("GOLD",           [], None, False, False ),
-    ParameterInfo("RED",            [], None, False, False ),
-    ParameterInfo("PURPLE",         [], None, False, False ),
-    ParameterInfo("LTBROWN",        [], None, False, False ),
-    ParameterInfo("ORANGE",         [], None, False, False ),
-    ParameterInfo("GREEN",          [], None, False, False ),
-    ParameterInfo("YELLOW",         [], None, False, False ),
-    ParameterInfo("DKGREEN",        [], None, False, False ),
-    ParameterInfo("CREAM",          [], None, False, False ),
-    ParameterInfo("BROWN",          [], None, False, False ),
-    ParameterInfo("WHITE",          [], None, False, False ),
-    ParameterInfo("LTBLUE",         [], None, False, False ),
-    ParameterInfo("GRAY",           [], None, False, False ),
-    ParameterInfo("DKBLUE",         [], None, False, False ),
-    ParameterInfo("BLACK",          [], None, False, False ),
-    ParameterInfo("PUSH_COLOUR",    [], None, False, False ),
-    ParameterInfo("POP_COLOUR",     [], None, False, False ),
+    ParameterInfo("BLUE",           [],    None, False, False ),
+    ParameterInfo("SILVER",         [],    None, False, False ),
+    ParameterInfo("GOLD",           [],    None, False, False ),
+    ParameterInfo("RED",            [],    None, False, False ),
+    ParameterInfo("PURPLE",         [],    None, False, False ),
+    ParameterInfo("LTBROWN",        [],    None, False, False ),
+    ParameterInfo("ORANGE",         [],    None, False, False ),
+    ParameterInfo("GREEN",          [],    None, False, False ),
+    ParameterInfo("YELLOW",         [],    None, False, False ),
+    ParameterInfo("DKGREEN",        [],    None, False, False ),
+    ParameterInfo("CREAM",          [],    None, False, False ),
+    ParameterInfo("BROWN",          [],    None, False, False ),
+    ParameterInfo("WHITE",          [],    None, False, False ),
+    ParameterInfo("LTBLUE",         [],    None, False, False ),
+    ParameterInfo("GRAY",           [],    None, False, False ),
+    ParameterInfo("DKBLUE",         [],    None, False, False ),
+    ParameterInfo("BLACK",          [],    None, False, False ),
+    ParameterInfo("COLOUR",         [P__], None, False, True  ),
```

Comment:
This is the `_NEWGRF_PARAMETERS` table for NewGRF string codes.
OpenTTD/OpenTTD#10687 seems to add a string code to OpenTTD and GS only.
The `_GS_PARAMETERS` table is further down. (`_OPENTTD_PARAMETERS` inherits all commands from the GS table)

Reply:
oups